docs/svelte-template-authoring.md

```diff
@@ -0,0 +1,68 @@
+# Svelte Template Authoring
+
+You'll first need to decide which of 2 types of templates is required, if your
```

Comment:
I think this sentence might be unfinished?

Reply:
Oops thanks!